tools/ci_connector_ops/ci_connector_ops/pipelines/utils.py

```diff
@@ -24,6 +24,7 @@
 
 if TYPE_CHECKING:
     from ci_connector_ops.pipelines.contexts import ConnectorContext
+    from github import PullRequest
```

Comment:
I was expecting some curl/API work in this PR.  I guess this just handles it!

Reply:
This [lib](https://github.com/PyGithub/PyGithub) is pretty good at wrapping the GitHub API for typed interaction.